Integrations/AWS_DynamoDB/AWS_DynamoDB.py

```diff
@@ -0,0 +1,1697 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+# flake8: noqa
+import boto3
+import json
+import datetime  # type: ignore
+from botocore.config import Config
+from botocore.parsers import ResponseParserError
+import urllib3.util
+
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+
+"""PARAMETERS"""
+AWS_DEFAULT_REGION = demisto.params().get('defaultRegion')
+AWS_ROLE_ARN = demisto.params().get('roleArn')
+AWS_ROLE_SESSION_NAME = demisto.params().get('roleSessionName')
+AWS_ROLE_SESSION_DURATION = demisto.params().get('sessionDuration')
+AWS_ROLE_POLICY = None
+AWS_ACCESS_KEY_ID = demisto.params().get('access_key')
+AWS_SECRET_ACCESS_KEY = demisto.params().get('secret_key')
+VERIFY_CERTIFICATE = not demisto.params().get('insecure', True)
+proxies = handle_proxy(proxy_param_name='proxy', checkbox_default_value=False)
+config = Config(
+    connect_timeout=1,
+    retries=dict(
+        max_attempts=5
+    ),
+    proxies=proxies
+)
+
+
+"""HELPER FUNCTIONS"""
+
+
+def safe_load_json(o):
+    kwargs = None
+    # Here we are checking the length of the entry to decide if it's a entry ID or json string.
+    if len(o) > 40:
```

Comment:
JSON could be less than 40 chars. Instead we should check 
`if (o.startswith('{') and o.endswith('}')) or (o.startswith('[') and o.endswith(']')):`